cfgov/apache/conf.d/redirects.conf

```diff
@@ -80,12 +57,6 @@ deposit-accounts-resources/.*|\
 mortgage-resources/.*\
 ) /compliance/compliance-resources/$1
 
-Redirect permanent /policy-compliance/notice-opportunities-comment/ /rules-policy/notice-opportunities-comment/
-Redirect permanent /policy-compliance/amicus/ /compliance/amicus/
-Redirect permanent /policy-compliance/enforcement/ /enforcement/
-Redirect permanent /policy-compliance/rulemaking/ /rules-policy/
-Redirect permanent /policy-compliance/guidance/ /compliance/
```

Comment:
I believe the redirects on line 83-87 were written to catch all of the sub-pages in each of these verticals. For instance, [`policy-compliance/guidance/amicus/`](https://www.consumerfinance.gov/policy-compliance/guidance/amicus/) redirects to `compliance/amicus/` because of this line.

Does you have a way of finding and accounting for all of the sub-pages that are affected by the pattern-matching in this file? (Sorry if I'm behind and you already addressed this.)

Reply:
I've added these back in as part of the latest commit. As far as I can tell Wagtail has no functionality for this. I'm not too concerned if there are other stray ones that intend to do this that we've missed, though -- as mentioned elsewhere, using server logs I'm planning to regularly check for redirect usage and remove (or re-add) redirects as needed accordingly. If any affected by this show up in that process, I'll incorporate them accordingly.